.circleci/config.yml

```diff
@@ -830,20 +845,23 @@ jobs:
       - run:
           name: Write pinned block number for cache key
           command: |
-            just print-pinned-block-number > ./pinnedBlockNumber.txt
+            echo <<parameters.fork_block_number>> > ./pinnedBlockNumber.txt
             cat pinnedBlockNumber.txt
           working_directory: packages/contracts-bedrock
       - restore_cache:
           name: Restore forked state
-          key: forked-state-contracts-bedrock-tests-upgrade-{{ checksum "packages/contracts-bedrock/pinnedBlockNumber.txt" }}
+          key: forked-state-contracts-bedrock-tests-upgrade-<<parameters.fork_op_chain>>-<<parameters.fork_base_chain>>-{{ checksum "packages/contracts-bedrock/pinnedBlockNumber.txt" }}
       - run:
           name: Run tests
           command: just test-upgrade
           environment:
             FOUNDRY_FUZZ_SEED: 42424242
             FOUNDRY_FUZZ_RUNS: 1
             FOUNDRY_PROFILE: ci
-            ETH_RPC_URL: https://ci-mainnet-l1-archive.optimism.io
+            ETH_RPC_URL: <<parameters.fork_base_rpc>>
+            FORK_OP_CHAIN: <<parameters.fork_op_chain>>
+            FORK_BASE_CHAIN: <<parameters.fork_base_chain>>
```

Comment:
The "base" chain nomenclature gets confusing since eventually we also want to run tests/ugprades against the Base L2. Can we call it "Fork L1 Chain" or similar? I know it technically is less precise because it could be an L2 Base chain for an L3, but we use the L1/L2 nomenclature in enough places and the intent is more clear that way